### PR TITLE
Finalise News Tags

### PR DESCRIPTION
**Pull request by baxgas**

### CODE REVIEW

**Review by gammaw — APPROVED** [review]
Wow! 

Couple of questions: 

- If it works for news, why can't we have them for shows too? 
- Any specific reason that you didn't add tags on `news `home?

**Comment by gammaw** [comment]
> If it works for news, why can't we have them for shows too?

@baxgas sorry and ignore it, just reminded me of this https://github.com/mmmnmnm/lahmacun_arcsi/issues/124

For my other question: I guess it's still valid, but it's just a usability improvement.

File: app/components/HeaderBlock.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <HeadNavigation />
     <div id="time">
       <div class="currenttime">
-        {{ timestamp }} <br>CET
+        {{ timestamp }} <br class="block xl:hidden">CET
```
[the remark this answers is not in the document]
Comment (baxgas) [reply app/components/HeaderBlock.vue:16]: @gammaw xl: is for large screens, where the menu can fit better :-)

[the remark this answers is not in the document]
Comment (gammaw) [reply app/components/HeaderBlock.vue:16]: So in those cases we hide the clock? Why is it not an issue for small screens?

**Pull request merged**